### ISSUE TITLE
Typescript declaration file support

### ISSUE THREAD

**Issue by JoshOY** [thelonious/kld-intersections#58]
Hi, I'm using using this library and found it splendid! But currently our project is written in typescript so is there any type declaration file (`.d.ts`) support for this library?

**Comment by Restuta** [comment]
@JoshOY I don't think so, but you can create one similar to https://github.com/DefinitelyTyped/DefinitelyTyped/pull/39899 that adds support for types of the fork of this project (https://github.com/signavio/svg-intersections) (I'd send a PR to that repo instead)
